.github/workflows/ci-static-build-test.yml

```diff
@@ -37,3 +37,12 @@ jobs:
     - name: Build (Ubuntu 20.04)
       run: |
         sudo -E tools/ci-build.sh
+        # Disabling checks until we find a better way to build a fully static binary.
```

Comment:
There is a way to build fully static binaries on Linux. It basically requires using different libc implementation than glibc as glibc will do `dlopen` for some features even if build statically and therefore it is never fully static ([SO thread](https://stackoverflow.com/questions/57476533/why-is-statically-linking-glibc-discouraged)). One option that I have seen successfully used in using the [Alpine Linux](https://www.alpinelinux.org/) that uses musl libc. I can even try making that work for p4c if desirable. But there is a counter-argument to that and that is that linking libc statically moves the critical compatibility interface from glibc, which is widely used and attempts to be forward-compatible to Linux kernel interface, which does not need to be so stable. I'm not sure how much concern this actually is as I don't have enough knowledge about Linux kernel interlace.

For most uses it should be sufficient to use static-sans-glibc and to build on a distro with the oldest glibc that can possibly be used. Note that using musl instead of glibc will likely bring some performance differences (and a small potential for different bugs).